canonicalwebteam/search/views.py

```diff
@@ -18,6 +23,7 @@ def build_search_view(
     template_path="search.html",
     search_engine_id="009048213575199080868:i3zoqdwqk8o",
     site_restricted_search=False,
+    request_limit="500/day",
```

Comment:
Eventually I decided to go for 500/day. The reason is because setting sth like `100/minute` the spammer will be blocked at request 101, but after 1 minute pause, it will be allowed to do requests again, adding up to thousands a day potentially, which at the end of the day could again reach the quota limit. Because Search API quota limit is set for the day, the requests should also limit on a per day basis I believe.

Reply:
OK happy to try this to start with.